src/olympia/files/tests/test_models.py

```diff
@@ -421,23 +421,42 @@ def setUp(self):
             AppVersion.objects.create(application=amo.FIREFOX.id,
                                       version=version)
 
-    def parse(self, addon=None, filename='extension.xpi'):
+    def parse(self, addon=None, filename='extension.xpi', minimal=None):
```

Comment:
I first wanted to suggest that `minimal` should be default `False` here too but then realized that disallows us from testing the actual default value of that method. 👍